crypto/fipsmodule/aes/internal.h

```diff
@@ -160,6 +170,9 @@ OPENSSL_INLINE int aes_hw_xts_cipher(const uint8_t *in, uint8_t *out, size_t len
                                       const uint8_t iv[16], int enc) {
   abort();
 }
+OPENSSL_INLINE int CRYPTO_is_xts_AESNI_capable(void) {
```

Comment:
FWIW (not sure we need this function), should prob return `0`?

Reply:
I matched the behaviour of the other functions, where this code should not be executed if the defines are set properly for detecting `HWAES_XTS`.

Reply:
I think the `*_capable` functions returns 0 normally, can be changed in https://github.com/awslabs/aws-lc/pull/469 if need be.

Reply:
Thank you. I changed it to return 0 and removed it in #469.